### PR TITLE
JS scroll animate anchor links header menu

### PR DESCRIPTION
**Pull request by Luanramos**
The header menu links softened with JS scroll animate, I thought the site would need something links menu.

### CODE REVIEW

**Comment by subhonjon** [comment]
Thanks

File: js/script.js [hunk 1]
```diff
@@ -597,4 +597,17 @@ app.filter('timeDiff', function() {
     };
```
Comment (subhonjon) [inline js/script.js]: Thanks

Comment (subhonjon) [inline js/script.js]: Thanks

**Comment by noraacl** [comment]
Not clear on the fix...

**Pull request closed without merging**